test_util/test_upgrade_vpc.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+import logging
+import os
+import random
+import string
+import sys
+
+import test_util.aws
+import test_util.cluster
+
+
+logging.basicConfig(format='[%(asctime)s|%(name)s|%(levelname)s]: %(message)s', level=logging.DEBUG)
+log = logging.getLogger(__name__)
+
+
+def main():
+    num_masters = 3
+    num_agents = 2
+    num_public_agents = 1
+    stack_name = 'upgrade-test-' + ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(10))
+
+    stable_installer_url = os.environ['STABLE_INSTALLER_URL']
```

Comment:
nit: could we generalize this to n step upgrades? (basically pass in a list of installer urls, first one is used to install, rest are used for upgrades?) Would help a lot for testing things like 1.7-open -> 1.8.3 -> 1.8.4 -> 1.9 upgrade path testing (Although that probably also would require a list of different config.yaml).